plugins/jobs/src/js/components/JobsForm.tsx

```diff
@@ -144,19 +150,122 @@ class JobModalForm extends React.Component<JobFormProps> {
     };
   }
 
+  getTabContent() {
+    const { jobSpec, errors, showAllErrors, i18n } = this.props;
+    const formOutput = this.getFormOutput(jobSpec);
+    const translatedErrors = translateErrorMessages(errors, i18n);
+    const pluginTabProps = {
+      errors: translatedErrors,
+      data: formOutput,
+      pathMapping: ServiceErrorPathMapping,
+      hideTopLevelErrors: !showAllErrors,
+      showErrors: showAllErrors,
+      onAddItem: this.handleAddItem,
+      onRemoveItem: this.handleRemoveItem
+    };
+
+    const tabs = [
+      // tslint:disable
```

Comment:
Why does tslint fight me over an array of jsx elements here??

Reply:
seems to be a mismatch in the ruleset of tslint and prettier, see cb6e7c39a59d705201dc0433ad38f22cc5f07f85.